### PR TITLE
Adding hCaptcha proxy component & Bumping core component version

### PR DESCRIPTION
**Pull request by navneet-ag**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by pankaj-parashar** [comment]
@navneet-ag, please resolve the conflict.

**Comment by navneet-ag** [comment]
> @navneet-ag, please resolve the conflict.

Resolved

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/hcaptcha/.content.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    cq:icon="shield"
+    jcr:primaryType="cq:Component"
+    jcr:title="Adaptive Form hCaptcha® (v1)"
+    sling:resourceSuperType="core/fd/components/form/hcaptcha/v1/hcaptcha"
+    componentGroup="Core Components Examples - Adaptive Form"/>
```
Comment (pankaj-parashar) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/hcaptcha/.content.xml]: Component group should be variable. Please refer https://github.com/adobe/aem-project-archetype/blob/develop/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/checkbox/.content.xml#L7

**Review by pankaj-parashar — CHANGES_REQUESTED** [review]

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/hcaptcha/.content.xml (hunk 1 above)
Comment (navneet-ag) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/hcaptcha/.content.xml]: Resolved

**Review by pankaj-parashar — APPROVED** [review]

**Pull request merged**